src/aluno/AlunoService.ts

```
@@ -1,17 +1,16 @@
 class AlunoService extends BaseService {
   constructor() {
     super('/usuarios/aluno')
   }
 
-  async save(
-    aluno: Aluno,
-  ): Promise<AxiosResponse<AuthenticationResponse>> {
-    return this.post(aluno)
+  async save(servidor: Aluno): Promise<void> {
+    const response = await this.post<{ token?: string }, Aluno>(servidor)
+    if (response && response.token) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)